iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/HvdcLineImpl.java

```diff
@@ -152,6 +157,11 @@ public HvdcLineImpl setActivePowerSetpoint(double activePowerSetpoint) {
         return this;
     }
 
+    @Override
+    public AbstractHvdcConverterStation<?> getConverterStation(Side side) {
+        return (side == Side.ONE) ? getConverterStation1() : getConverterStation2();
```

Comment:
Is it necessary as it is already defined as a default method in HvdcLine interface?

Reply:
If you want to remove one day, the default implementation in `HvdcLine` interface, you have to implement this method in all the sub classes. I can also consider there is only one implementation of IIDM for now.
What do you prefer?